default.nix

```diff
@@ -0,0 +1,13 @@
+{ pkgs ? import <nixpkgs> {} }:
+with pkgs;
+buildGoModule {
+  pname = "hey";
+  version = "0.1.3";
```

Comment:
please update to the newest release

Reply:
Done. Per convention, I've used `version = "git"` here, because it's not pinned to any particular version.